go.mod

```diff
@@ -0,0 +1,5 @@
+module knative.dev/eventing-integrations
+
+go 1.23.1
+
+require knative.dev/hack v0.0.0-20241010131451-05b2fb30cb4d
```

Comment:
We only need `go.mod` to enable our hack scripts.

Reply:
sounds good, @dsimansk 

Thanks a lot for getting into this